### ISSUE TITLE
GPU acceleration issue

### ISSUE THREAD

**Issue by 777Beta** [leoliuf/MRiLab#35]
Hi,

I'm running MRiLab v1.3 on a computer equipped with NVDA Tesla V100 GPU and CUDA 11.0. As long as I lunch MRiLab in MATLAB the attached message pops up, and I cannot run simulation with GPU. I noticed the user manual mentioned development environment with CUDA 7.0.28. My question is whether my GPU and CUDA version is too advanced for MRiLab? CUDA 7.0 cannot support my GPU.
Thanks much!

![MRiLab GPU error](https://user-images.githubusercontent.com/72461105/95231334-02b66800-07d1-11eb-8085-46d52ea6c6ad.jpg)

**Comment by leoliuf** [comment]
Hi! You need to recompile the CUDA code against the right version you are currently using. The shipped precompiled libraries are apparently not working with the latest CUDA versions.
